src/tools/info/proto_info.c

```diff
@@ -93,25 +93,75 @@ static void print_resource_usage(const resource_usage_t *usage_before,
     printf("#\n");
 }
 
-ucs_status_t print_ucp_info(int print_opts,
-                            ucs_config_print_flags_t print_flags,
-                            uint64_t ctx_features,
-                            const ucp_ep_params_t *base_ep_params,
-                            size_t estimated_num_eps, size_t estimated_num_ppn,
-                            unsigned dev_type_bitmap, const char *mem_size)
+static void listener_accept_callback(ucp_ep_h ep, void *arg)
+{
+    ucp_ep_h *server_ep = (ucp_ep_h*)arg;
+
+    *server_ep = ep;
+}
```

Comment:
minor: really strange function...
why not just ```*(ucp_ep_h*)arg = ep```

Reply:
fixed